engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionBuilder.scala

```diff
@@ -285,16 +289,18 @@ final class SuggestionBuilder[A: IndexedSource](
     module: QualifiedName,
     name: String,
     location: Location,
+    doc: Option[String],
     typeSignature: Option[TypeSignatures.Metadata]
   ): Suggestion.Local = {
     val typeSig            = buildTypeSignatureFromMetadata(typeSignature)
     val (_, returnTypeDef) = buildFunctionArguments(Seq(), typeSig)
     Suggestion.Local(
-      externalId,
-      module.toString,
-      name,
-      buildReturnType(returnTypeDef),
-      buildScope(location)
+      externalId    = externalId,
+      module        = module.toString,
+      name          = name,
+      returnType    = buildReturnType(returnTypeDef),
+      scope         = buildScope(location),
+      documentation = doc
```

Comment:
Is there a metric measuring how much suggestion DB data is being sent to the IDE? I'd like to know the amount of data sent before and after this change.

Reply:
There is a way to create a file-based database instead of in-memeory, and measure the file size https://github.com/enso-org/enso/blob/0465aa3080baa109ea616348505a517d2e6014e7/engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala#L117

Right now the standard library is around 3MB. Most of it are docstrings.